hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -626,10 +610,10 @@ public static DBStore loadDB(OzoneConfiguration configuration, File metaDir,
         .setPath(Paths.get(metaDir.getPath()))
         .setMaxFSSnapshots(maxFSSnapshots)
         .setEnableCompactionDag(enableCompactionDag)
-        .setCreateCheckpointDirs(createCheckpointDirs);
+        .setCreateCheckpointDirs(createCheckpointDirs)
+        .setMaxNumberOfOpenFiles(maxOpenFiles);
```

Comment:
This will set maxOpenFiles even for Active File System. We only want this property to be set for the Snapshot db and OM checkpoint db.

Reply:
For Active File System, this value is set to `-1` which means infinite.

https://github.com/apache/ozone/blob/8f4ea86ada323d61eb87f337627e5f089682b1f8/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L580

 Refer to https://github.com/apache/ozone/pull/6954.